core/controller/src/main/scala/org/apache/openwhisk/core/controller/Actions.scala

```diff
@@ -253,7 +247,7 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
       'result ? false,
       'timeout.as[FiniteDuration] ? WhiskActionsApi.maxWaitForBlockingActivation) { (blocking, result, waitOverride) =>
       entity(as[Option[JsObject]]) { payload =>
-        getEntity(WhiskActionMetaData.get(entityStore, entityName.toDocId), Some {
+        getEntity(WhiskActionMetaData.resolveActionAndMergeParameters(entityStore, entityName), Some {
```

Comment:
@style95 the action is resolved here after it's dispatched.

Reply:
ok, this is the similar way which is taken for web action.